playa/page.py

```diff
@@ -1107,54 +1090,64 @@ def __iter__(self) -> Iterator[GlyphObject]:
             )
             self._next_tstate = tstate
             return
-        assert self.ctm is not None
-        # Extract all the elements so we can translate efficiently
-        a, b, c, d, e, f = mult_matrix(tstate.line_matrix, self.ctm)
-        # Pre-determine if we need to recompute the bound for rotated glyphs
-        corners = b * d < 0 or a * c < 0
-        # Apply horizontal scaling
-        scaling = tstate.scaling * 0.01
+        vert = font.vertical
+        # Apply horizontal scaling if font is horizontal
+        scaling = 1 if vert else tstate.scaling * 0.01
         charspace = tstate.charspace * scaling
         wordspace = tstate.wordspace * scaling
-        vert = font.vertical
+        # PDF 1.7 sec 5.5.3 seems to suggest that the displacement is always either
+        # horizontal or vertical. Does that imply Type 3 font matrix preserves x axis?
```

Comment:
I was just asking myself the same question in the context of https://github.com/docling-project/docling-parse/pull/113 ... the answer is probably "write a PDF with a Type3 font whose matrix contains rotation and see what Acrobat thinks about it"

Reply:
I think this is just undefined behaviour, but I don't have an Acrobat or an Apple to test those.  pdf.js does something sensible:

![image](https://github.com/user-attachments/assets/b6e063e6-455c-4c47-ae8a-0abfb9ce44ff)

whereas Poppler does something considerably less sensible:

![image](https://github.com/user-attachments/assets/af97ca4e-cb1c-43f8-88c3-34d65d35d743)

Reply:
in both cases the bbox is *not* the actual bound of the characters but, well, something else (maybe the transformed corners? maybe?):
![image](https://github.com/user-attachments/assets/e1eb7225-6b8c-408b-97fd-6ef278578088)
![image](https://github.com/user-attachments/assets/f5b30111-e073-4971-9cb5-8e4f78097e30)